HOWTO/DOCUMENTATION.md

```diff
@@ -136,6 +139,288 @@ It is also possible to customize the groups for an application by adding a `grou
 `docs.exs`. See the [system application's docs.exs](../system/doc/docs.exs) for an example on how
 to do that.
 
+## Documentation style rules
+
+When writing documentation for the Erlang/OTP project, the following
+rules should be followed. The documentation has been written by a lot
+of different people, with their own writing style, that may or may not
+be following this style guide. When in doubt, follow the style that is
+used in the document you are currently editing in order to keep the
+same style throughout that document.
+
+### Grammar rules and guidelines
+
+American English is to be used in all documents.
+
+#### Word list
+
+* behavior - Should be spelled this way because it is American
+  English. However, the module attribute is named "-behaviour.".
+
+* back-end - Not "backend"
+
+* boot script - ".script file", a script specifying which code to load
+  and which applications to start. The binary version is called a boot
+  file, see below.
+
+* boot file - ".boot file", an argument for the command line flag -boot.
+
+* code path - Not "search path"
+
+* command line arguments - Everything after `erl` on the command line.
+
+* command line flags - Flags after `erl` on the command line, for
+  example, `-boot`. Subset of "command line arguments"
+
+* (application) configuration parameter - Values that can be set in a
+  `.config` file, or from Erlang shell or command line. Not
+  "(application) variable".
+
+* Erlang, Erlang/OTP, OTP - "Erlang" is the language. "Erlang/OTP" is
+  the product and should normally be referred to as such. "OTP" is the
+  project. "OTP" can also in some cases be used to refer to the parts
+  of the product that are completely decoupled from the Erlang
+  part. Cf. 'OTP Design Principles', a part of Erlang/OTP, but you can
```

Comment:
should `Cf.` acronym be used?
Section "Contractions and abbreviation" does not recommend only certain acronyms - why not using acronyms at all?